utils/logging.js

```diff
@@ -75,11 +85,12 @@ const skipCommon = (req, res) => {
   const { method, path, query } = req;
   const { statusCode } = res;
   const ref = req.get("referer") || req.get("referrer");
-  const referrer = ref ? new URL(ref) : null;
```

Comment:
saw someone send a request with `referfer: 'example.com'` which crashed the server

Reply:
I want a referfer tattoo.